### ISSUE TITLE
Inconsistencies in LDAP documentation

### ISSUE THREAD

**Issue by OlivierBondu** [firefly-iii/firefly-iii#5073]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

It looks like the [`.env.example`](https://raw.githubusercontent.com/firefly-iii/firefly-iii/main/.env.example) I was referring to for setting up LDAP is in fact outdated compared to the main documentation at https://docs.firefly-iii.org/firefly-iii/advanced-installation/authentication/

in particular the name of all variables are now prefixed with `AD` and some were renamed.

### Expected behaviour

Up to date [`.env.example`](https://raw.githubusercontent.com/firefly-iii/firefly-iii/main/.env.example)  (alhough digging around the relevant documentation already exist)

### Debug information

NA

### Additional info

Thanks a lot for the excellent work!

**Comment by JC5** [comment]
Feel free to submit a PR to patch the bad variable names. I am working on a new version with an updated LDAP library. Unfortunately, that will mean the variable names will change *again*. However, the matching documentation is already ready.

**Comment by OlivierBondu** [comment]
The problem is I am not sure who to trust between the example env file or the documentation... which one is the source of truth ? The env file seem more recent.

On the other hand, if you are working on LDAP stuff, it may not be worth a PR depending on the timeline ?

In any case, happy to help if I can

**Comment by JC5** [comment]
True,although the .env.example file will always be the most correct. I'll label this as an enhancement and close it with the new release.
